### ISSUE TITLE
[Feature Request] Add filter to logs tab

### ISSUE THREAD

**Issue by denil-ct** [mockoon/mockoon#1109]
Add a filter bar to logs tab similar to routes tab. Its very hard to read through all the logs, as the pane is usually not wide enough to see the complete endpoint. It also helps to focus on a specific api and clean up all the clutter.

**Comment by 255kb** [comment]
I will close this issue as a duplicate of #137 
But I agree, it's annoying and would be very useful.

**Comment by denil-ct** [comment]
@255kb I went through that issue and that was more concerned on searching over everything, whereas this is more focused on the endpoint alone.

**Comment by 255kb** [comment]
@denil-ct ok, I see. Wouldn't it make sense to filter by any value inside the path/request/response? 
For example, in Chrome dev tools, when you filter the requests in the network tab it will search in more than the path. 

Also, Mockoon's route filter is already looking in the path, method and documentation.

**Comment by denil-ct** [comment]
@255kb I agree, that would be the ideal solution, but that would also take more development time.
Meanwhile this could be comparatively simpler. When we want to extend search results to request/response it can be done easier.
Basically, instead of doing it all in one go, I suggest we do step by step.

Is there a plan to do search on response body, headers in the routes tab to reach parity on both tabs?

**Comment by 255kb** [comment]
> Is there a plan to do search on response body, headers in the routes tab to reach parity on both tabs?

Now that I think about it, no there is no such plan. And the route filtering is some sort of hack looking into each HTML menu element's text. 
If we do a on the body, headers, etc, it should probably be some sort of global search field that allows to reach any kind of entity. (A bit like a search feature on a website)

You convinced me, I will put this on the roadmap as a simple filter for now.

**Comment by denil-ct** [comment]
Awesome, thanks @255kb . 
I am by no means a front end developer , but I will try to add a filter to the logs tab as a learning exercise. If its successful I will raise a PR.

**Comment by 255kb** [comment]
Sure, go for it! 
You can follow how it was done for the routes menu. 
The basic idea is that there is an Angular attribute (directive) on each menu element, that receives an observable. The observable is basically the filter input content. 

Let me know if you have question. You can reach me out on [Discord](https://discord.com/invite/JmTBk59MTb).

**Comment by 255kb** [comment]
📦 This was added in [v8.1.0](https://mockoon.com/releases/8.1.0/)
